### DISCUSSION TITLE
4.2 beta .. is a demo available?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#18047] [General]
I see FR's being submitted for the 4.2 beta. 
The beta site : https://beta-demo.netbox.dev doesn't resolve.
I'm just wondering if a 4.2 beta demo will be available for testing?

**Comment by jeremystretch** [comment]
v4.2 beta has not been released yet. It's due to be released December 2.

**Reply by ghost** [reply]
sounds good. thanks.
